### ISSUE TITLE
Components in storybook are not loading

### ISSUE THREAD

**Issue by davidkudera** [adobe/spectrum-web-components#1560]
### Expected Behaviour
After cloning the repository, `yarn storybook` should work.

### Actual Behaviour
The `yarn storybook` command opens the browser, but only the layout without any components is loaded. It also seems like something is triggering the storybook watcher on init multiple times. In the latest working version (b78ec02c445ab39c5dd5ebb79eb1b8265f471090) there were no changes detected on init. But I'm not sure if these two problems are related.

### Reproduce Scenario (including but not limited to)
Local development

#### Steps to Reproduce
```bash
git clone https://github.com/adobe/spectrum-web-components.git
cd spectrum-web-components
yarn
yarn storybook
```

#### Platform and Version
* Windows 10 - tried with git bash, Powershell and WSL
* Chrome 91.0.4472.114 - with and without incognito 
* Firefox 89.0.1

#### Logs taken while reproducing problem
**CLI:**

These 2 logs are repeating for about one minute:

```bash
[14:55:36] File change detected. Starting incremental compilation...

[14:55:36] Found 0 errors. Watching for file changes.
```

```bash
Web Dev Server started...

  Root dir: C:\Projects\spectrum-web-components
  Local:    http://localhost:8000/
  Network:  http://10.13.18.4:8000/
```

**Browser:**
```
Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "video/mp2t". Strict MIME type checking is enforced for module scripts per HTML spec.    | index.ts:1
```

**Comment by Westbrook** [comment]
Fixed by #1561 🤞🏼

**Comment by davidkudera** [comment]
@Westbrook Yay! It works now, thank you 🍻
